target/linux/lantiq/patches-4.4/0154-lantiq-pci-bar11mask-fix.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/arch/mips/pci/pci-lantiq.c b/arch/mips/pci/pci-lantiq.c
+index 6a15dbd..c10ca7d 100644
+--- a/arch/mips/pci/pci-lantiq.c
++++ b/arch/mips/pci/pci-lantiq.c
+@@ -62,6 +62,8 @@
+ #define ltq_pci_cfg_w32(x, y)	ltq_w32((x), ltq_pci_mapped_cfg + (y))
+ #define ltq_pci_cfg_r32(x)	ltq_r32(ltq_pci_mapped_cfg + (x))
+ 
++extern u32 max_low_pfn;
++
+ __iomem void *ltq_pci_mapped_cfg;
+ static __iomem void *ltq_pci_membase;
+ 
+@@ -82,13 +84,24 @@ static struct pci_controller pci_controller = {
+ 	.io_offset	= 0x00000000UL,
+ };
+ 
++static u32 round_up_to_next_power_of_two(u32 x)
++{
```

Comment:
linux/log2.h has a #define roundup_pow_of_two(n) helper. please use this instead